From: Stig Sandbeck Mathisen <ssm@debian.org>
Date: Sun, 6 Apr 2014 23:46:43 +0200
Subject: Do not require rubygems

---
 bin/facter | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/bin/facter b/bin/facter
index 0616157..a25fa03 100755
--- a/bin/facter
+++ b/bin/facter
@@ -1,16 +1,5 @@
 #!/usr/bin/env ruby
 
-# Bundler and rubygems maintain a set of directories from which to
-# load gems. If Bundler is loaded, let it determine what can be
-# loaded. If it's not loaded, then use rubygems. But do this before
-# loading any facter code, so that our gem loading system is sane.
-if not defined? ::Bundler
-  begin
-    require 'rubygems'
-  rescue LoadError
-  end
-end
-
 require 'facter/application'
 
 Facter::Application.run(ARGV)
